share/picongpu/examples/LaserWakefield/include/picongpu/simulation_defines/param/laser.param

```diff
@@ -45,7 +59,7 @@ namespace picongpu
         namespace SI
         {
             /** unit: meter */
-            constexpr float_64 WAVE_LENGTH_SI = 0.8e-6;
+            constexpr float_64 WAVE_LENGTH_SI = UI_MACRO_LASER_WAVELENGTH;
```

Comment:
Can you please add the macros to the defined of the other lasers below as well? This would allow us to switch the laser profile later on in `components.param` as well.

Reply:
> you adjusted all lasers in laser.param but only Gaussian is used. So please revert your changes for the other lasers or add a switch between the laser profile used.

sorry for that first review comment. you did it right in the first place, please keep it for all lasers.